.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+  - "10.15"
+
+install:
+  - npm install
+
+branches:
+  only:
+    - master
+
+before_sript:
+  - npm install -g @angular/cli
+  - npm install
```

Comment:
To `npm install` to się w sumie dzieje 2x?

Reply:
`npm install -g @angular/cli` - instaluje globalnie Angulara
`npm install` - instaluje całą resztę zdefiniowaną w package.json

Reply:
Tak jest, to wiem - miałam na myśli to w linii 6 właśnie

Reply:
Jeszcze w sumie odkryłam, że istnieje coś takiego jak `npm ci`, które używa `package-lock.json` i robi jakieś tam rzeczy pod CI specjalnie: https://docs.npmjs.com/cli/ci.html